source/index.html.erb

```diff
@@ -38,28 +38,36 @@ products:
       link: "/product-stories/petitions"
 
   client_logos:
-    - name: "DPD"
-      image: "/assets/images/homepage/logo-dpd.gif"
+    - name: "Deliveroo"
+      image: "/assets/images/homepage/logo-deliveroo.gif"
     - name: "Which?"
       image: "/assets/images/homepage/logo-which.gif"
     - name: "Sky"
       image: "/assets/images/homepage/logo-sky.gif"
     - name: "Not on the High Street"
       image: "/assets/images/homepage/logo-not-on-the-highstreet.gif"
-    - name: "Funding Circle"
-      image: "/assets/images/homepage/logo-funding-circle.gif"
+    - name: "Southwark CCG"
+      image: "/assets/images/homepage/logo-southwark-ccg.jpg"
     - name: "Guy's and St Thomas' NHS Foundation Trust"
       image: "/assets/images/homepage/logo-gstt.gif"
     - name: "Apps for Good"
       image: "/assets/images/homepage/logo-apps-for-good.gif"
-    - name: "Government Digital Service"
-      image: "/assets/images/homepage/logo-gds.jpg"
+    - name: "Government UK"
```

Comment:
Worth checking with @GramMcCram I think, but "Government UK" doesn't sound right to me.  We'd talk about "the UK Government" or "GOV.UK" but not "Government UK".

Reply:
Oh! I was too slow for this PR as, but it's still worth checking I reckon!

Reply:
I agree with @h-lame - 'GOV.UK' would be a more suitable name for this tile.